src/ext/backtrace.c

```diff
@@ -44,10 +58,8 @@ void ddtrace_install_backtrace_handler(TSRMLS_D) {
 
     static int handler_installed = 0;
     if (!handler_installed) {
```

Comment:
Shouldn't this be:

```suggestion
    if (!backtrace_globals.handler_installed) {
```

And deleting the `static int handler_installed = 0;` statement? :)